core/pom.xml

```diff
@@ -160,5 +160,19 @@
             <version>1.5.2 </version>
             <scope>test</scope>
         </dependency>
+        <!-- https://mvnrepository.com/artifact/org.apache.poi/poi -->
+		<dependency>
+		    <groupId>org.apache.poi</groupId>
+		    <artifactId>poi</artifactId>
+		    <version>4.0.1</version>
+		    <optional>true</optional>
+		</dependency>
+		<!-- https://mvnrepository.com/artifact/org.apache.poi/poi-ooxml -->
+		<dependency>
```

Comment:
can you change the tabs to spaces so the indentation aligns with the other dependencies?

Reply:
same with the other files. I notice they have a mix of tabs and spaces as well